src/enkf/gridinfo_wrf.f90

```diff
@@ -211,7 +213,9 @@ subroutine getgridinfo_arw(fileprefix)
     ! Allocate memory for global arrays
     if(.not. allocated(lonsgrd)) allocate(lonsgrd(npts))
     if(.not. allocated(latsgrd)) allocate(latsgrd(npts))
+    if(.not. allocated(taper_vert)) allocate(taper_vert(nlevs))
     if(.not. allocated(logp))    allocate(logp(npts,nlevs_pres))
+    taper_vert = one
```

Comment:
Difference precisions.  `taper_vert` is `r_single`.  `one` is `r_kind`

Reply:
done